jujugui/static/gui/src/app/assets/css/_mixins.scss

```diff
@@ -1,3 +1,6 @@
+@import './plugins/cloud-vanilla-theme/scss/global-settings';
```

Comment:
Are imports deduped?

Reply:
The imports only import variables, so when compiled it only uses the values as required.